src/components/SharedItem/SharedItem.test.js

```diff
@@ -1,27 +1,45 @@
 import React from "react";
 import SharedItem from "./SharedItem";
 import { renderWithReduxAndRouter } from "../../services/@testing-library-helpers";
-import { mockWork } from "../../testing-helpers/mock-work";
+import { act } from "@testing-library/react";
+import { mockWork } from "testing-helpers/mock-work";
+import { mockWorkVideo } from "testing-helpers/mock-work-video";
 
-xdescribe("SharedItem component", () => {
-  it("renders without crashing", () => {
+describe("SharedItem component", () => {
+  it("renders without crashing", async () => {
+    const promise = Promise.resolve();
     const { getByTestId } = renderWithReduxAndRouter(
       <SharedItem work={mockWork} />
     );
     expect(getByTestId("shared-item"));
+    await act(() => promise);
```

Comment:
Why do we need this, or what is it helping with?

Reply:
@adamjarling These tests were being skipped for some reason and I discovered when turning them back on that I was getting a warning after they passed:

```
 When testing, code that causes React state updates should be wrapped into act(...):

  act(() => {
    /* fire events that update state */
  });
  /* assert on the output */
```

Instead of ignoring, I looked around the internet and came across this write-up by Kent Dodds https://kentcdodds.com/blog/fix-the-not-wrapped-in-act-warning#an-alternative-waiting-for-the-mocked-promise.

I think the idea here is that we are trying to let the test know that it's complete and we do that by mocking a promising and following through with it via `act()`.

Reply:
@mathewjordan Just pushed up a commit which gives an example of how we can write tests which rely upon elements appearing in the DOM asynchronously, or delivered through a promise or something not immediately rendered.  

Let me know if you have any questions?   Let's update the rest of the tests to match this pattern.